api/src/services/SolrIndexer.ts

```diff
@@ -110,8 +110,8 @@ class SolrIndexer {
 
         // Process parent data (note that hierarchyParents makes some special exceptions for VuFind;
         // fedoraParents exactly maintains the hierarchy as represented in Fedora):
-        const hierarchyParents: Array<FedoraData> = [];
-        const fedoraParents: Array<FedoraData> = [];
+        const hierarchyParents: Array<FedoraDataCollection> = [];
```

Comment:
What is the difference between these precents?

Reply:
The comment above is intended to explain -- VuFind (our front end system) has a specific way of representing hierarchical data, so hierarchyParents follows VuFind's rules. The fedoraParents array is a raw representation of the data in Fedora, without applying VuFind-specific rules.

Reply:
What about vuFindParents?

Reply:
I think renaming that variable could make sense, but I think it's beyond the scope of this PR since I don't want to change the internals of the classes at the same time that I'm renaming them. I will open a separate PR as a follow-up when this one is merged.

Reply:
See #125.